starter-code/basic-algorithms.js

```diff
@@ -1,7 +1,52 @@
 // Names and Input
-
+var hacker1 = "Ryan";
+var hacker2 = "Diego";
+console.log("The driver's name is " + hacker1);
+console.log("The navigator's name is " + hacker2);
 
 //Conditionals
+if(hacker1.length === hacker2.length){
+  console.log("Wow, you've both got eually long names, " + hacker1.length + "characters!!");
+}
+else if(hacker1.length > hacker2.length){
+  console.log("The Driver has the longest name, it has " + hacker1.length + " characters.");
+}
+else if(hacker1.length < hacker2.length){
+  console.log("Yo, navigator got the longest name, it has " + hacker2.length + " characters.");
+}
+
+for(var i = 0; i < hacker1.length; i++){
+  console.log(hacker1[i].toUpperCase() + " ");
+}
```

Comment:
the for loop is printing out each letter on a separate line. In order to get it to print out in one line you would try putting the for loop within a function and have a variable that would return or get logged at the end of the function.